src/pages/payment.tsx

```diff
@@ -0,0 +1,183 @@
+    const sessionUrl = await createStripeSessionMutation(orderInfo)
+      .unwrap()
+      .then((res) => res.sessionUrl);
+    window.location.href = sessionUrl;
+  };
+
+  return (
+    <>
+      <NextHeadSeo
+        title="Payment response page - Courtcanva"
+        description="Payment success or failure page on CourtCanva"
+        canonical="http://www.courtcanva.com/payment"
+      />
+      <Box
+        height="calc(100vh - 72px)"
+        width="100vw"
+        position="fixed"
+        top="72px"
+        color="brand.primary"
+        minWidth="800px"
+        minHeight="600px"
+      >
+        <Flex
+          flexDirection="column"
+          justifyContent="center"
+          alignItems="center"
+          gap={{ base: "2rem", md: "2.5rem", lg: "3rem" }}
+          marginTop="7vh"
+        >
+          <Flex
+            justifyContent="center"
+            alignItems="center"
+            gap={{ base: "0.5rem", md: "0.9rem", lg: "1.3rem" }}
+          >
+            <Box
+              fontSize={{ base: "2.4rem", md: "3rem", lg: "3.6rem" }}
+              color={status === PageStatus.SUCCESS ? "brand.secondary" : "#d64543"}
+            >
+              {status === PageStatus.SUCCESS ? <IoIosCheckmarkCircleOutline /> : <BiErrorCircle />}
+            </Box>
+            <Text fontSize={{ base: "1.6rem", md: "2rem", lg: "2.4rem" }} fontWeight="600">
+              {status === PageStatus.SUCCESS ? "Payment Successful" : "Payment Failed"}
+            </Text>
+          </Flex>
+          <Flex
+            boxShadow="4px 4px 8px rgba(0, 0, 0, 0.25), -4px -4px 8px #EBECF0"
+            width="90vw"
+            height="45vh"
+            borderRadius="1rem"
+            flexDirection="column"
+            alignItems="center"
+            justifyContent="space-around"
+            paddingTop="1rem"
+            minWidth="720px"
+          >
+            <Text
+              fontSize={{ base: "1.2rem", md: "1.5rem", lg: "1.8rem" }}
+              fontWeight="600"
+              lineHeight="1"
+            >
+              {status === PageStatus.SUCCESS ? "Thank you!" : "Something went wrong!"}
+            </Text>
+            <Text
+              fontSize={{ base: "0.7rem", md: "0.9rem", lg: "1.1rem" }}
+              fontWeight="600"
+              paddingBottom="1rem"
+            >
+              {status === PageStatus.SUCCESS
+                ? "You have successfully paid the deposit. Our staff will contact you soon."
+                : "Your payment could not be processed. Please try again."}
+            </Text>
+            <Box height="0.1rem" background="#D3D3D3" width="calc(100% - 30vw)"></Box>
+            <OrderInfoCard paymentResPage={paymentInfo} />
+          </Flex>
+          <Flex justifyContent="center" alignItems="center" gap="3rem">
+            <Button variant="shareBtn" padding="1rem 1.5rem" fontSize="1.2rem">
+              Check My Order
```

Comment:
A minor one, would you please make this "Check My Order" button and the "Try again" button consistent in writing format? :)

Reply:
> A minor one, would you please make this "Check My Order" button and the "Try again" button consistent in writing format? :)

fixed